dev-python/django-prometheus/django-prometheus-2.1.0.ebuild

```
@@ -1,9 +1,9 @@
-# Copyright 1999-2020 Gentoo Authors
+# Copyright 1999-2021 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=7
 
-PYTHON_COMPAT=( python3_7 )
+PYTHON_COMPAT=( python3_{7..9} )
 inherit distutils-r1
 
 DESCRIPTION="Library to export django metrics for prometheus"
```

DEPEND needed? Also:
```suggestion
DESCRIPTION="Library to export Django metrics for Prometheus"
```

---

Done